tools/build/cross-build/progname.c

```diff
@@ -36,7 +36,6 @@
 #include <sys/cdefs.h>
 #include <stdlib.h>
 
-#ifdef __GLIBC__
```

Comment:
or this

Reply:
I guess because tools/build/Makefile only builds it on Linux, so it was previously a stupid ifdef, and now is unhelpful for musl.

Reply:
I feel like this file might have been compiled for macos in the past so the ifdef was there to only use it for Linux. Dropping it looks good.